chipy_org/apps/meetings/models.py

```diff
@@ -249,8 +300,13 @@ class RSVP(CommonModel):
     last_name = models.CharField(max_length=MAX_LENGTH, blank=True, null=True)
     first_name = models.CharField(max_length=MAX_LENGTH, blank=True, null=True)
     email = models.EmailField(max_length=255, blank=True, null=True)
+
     meeting = models.ForeignKey(Meeting, on_delete=models.CASCADE)
-    response = models.CharField(max_length=1, choices=RSVP_CHOICES)
+
+    response = models.CharField(max_length=1, choices=RSVP_RESPONSE, default=YES)
+    status = models.CharField(max_length=1, choices=RSVP_STATUS, blank=True, null=True)
+    venue = models.CharField(max_length=1, choices=RSVP_VENUE, default=IN_PERSON)
+
```

Comment:
My general thoughts on these is that I'm not a big fan of single character database values. We have a scale where optimizing for this isn't important (we pay per record way before we would pay per MB) and it makes intuiting things about the data more challenging.

Reply:
fair enough - the single chars do become easier to implement in Django 3.2 - but you do run into nested classes ...

Reply:
I made it 50 chars with a full description.